pages/cloudflare/caching.mdx

```diff
@@ -45,3 +45,28 @@ The binding name used in your app's worker is `NEXT_CACHE_WORKERS_KV`.
   ]
 }
 ```
+
+#### 3. Configure the cache
+
+In your project's OpenNext config, enable the KV cache and set up a queue.
+
+The memory queue will send revalidation requests to a page when needed, and offers support for de-duplicating requests on a per-isolate basis.
```

Comment:
Maybe add an explicit not that there could still be duplicate requests (i.e. high traffic and different regions)

Reply:
```suggestion
The memory queue will send revalidation requests to a page when needed, and offers support for de-duplicating requests on a per-isolate basis. There might still be duplicate requests under high traffic or across regions.
```